ui/src/views/offering/ImportBackupOffering.vue

```diff
@@ -167,7 +167,7 @@ export default {
             params[key] = input
           }
         }
-        params.allowuserdrivenbackups = values.allowuserdrivenbackups ? values.allowuserdrivenbackups : true
+        params.allowuserdrivenbackups = values.allowuserdrivenbackups == null ? true : values.allowuserdrivenbackups
```

Comment:
This may change the default behaviour, needs checking

Reply:
This change was made to fix the reported bug of `Allow User Driven Backups` parameter.

In the old code, when user set this parameter as `false`, this check, as it is a ternary operation, it always ended up sending the parameter as `true`.

Reply:
@SadiJr You can edit the following instead of your condition:
1. `v-decorator="['allowuserdrivenbackups', { initialValue: true }]"` in template `form-item`
2. `params.allowuserdrivenbackups = values.allowuserdrivenbackups` in `params`

Reply:
@utchoang:
Done, and working fine. Thanks for the suggestion. 

@rhtyd:
Can you test and review the new code? The default behaviour, for my tests, don't changed.